src/js/components/DeploymentsListComponent.jsx

```diff
@@ -184,6 +191,11 @@ var DeploymentListComponent = React.createClass({
               Error fetching deployments. Refresh to try again.
             </td>
           </tr>
+          <tr className={unauthorizedClassSet}>
+            <td className="text-center text-danger" colSpan="6">
+              Error fetching deployments. Unauthorized access.
```

Comment:
I really don't see a performance concern in this case, so let's definitely re-discuss this. I'd really like to avoid the proliferation of elements that could be easily re-used, so if you prefer to do that in a separate refactoring PR, fine. I will create an issue to keep track.